Source/WPF/MyMoney/MainWindow.xaml.cs

```diff
@@ -274,6 +274,23 @@ private void OnSettingsChanged(object sender, System.ComponentModel.PropertyChan
             }
         }
 
+        private void ApplyDefaultCurrency()
+        {
+            // Start with the default USA currency
+            var countryCurrencyCode = "en-USD";
+            decimal convertionRate = 1;
+
+            // Lookup the users App default display currency
+            Currency c = this.myMoney.Currencies.FindCurrency(this.databaseSettings.DisplayCurrency);
+            if (c != null)
+            {
+                countryCurrencyCode = c.CultureCode;
+                convertionRate = c.Ratio;
+            }
+            this.myMoney.Rate = convertionRate;
+            this.myMoney.CultureInfo = StringHelpers.TryToGetCulureDefaultBackToUS(countryCurrencyCode);

```

Comment:
I'd prefer this to not be a new field on myMoney but instead do this:

`CultureInfo.CurrentUICulture = ...`

And this should be decoupled from currency selection and should not be a column in the currency database, instead it should be an option on the AppSettings for the entire app which leads to then being able to localize the entire app, all the strings, etc. Then we can do a full french version which would be awesome.

Reply:
How about we do this as a separate effort. Localize the entire App?